x-pack/plugins/ml/public/application/services/ml_api_service/index.ts

```diff
@@ -53,6 +54,33 @@ import { savedObjectsApiProvider } from './saved_objects';
 import { trainedModelsApiProvider } from './trained_models';
 import { notificationsProvider } from './notifications';
 
+export interface SecurityHasPrivilegesDisabledResponse {
+  securityDisabled: boolean;
+  upgradeInProgress: boolean;
+}
+
+export interface ExtendedSecurityHasPrivilegesResponse {
+  hasPrivileges: estypes.SecurityHasPrivilegesResponse;
+  upgradeInProgress: boolean;
+}
```

Comment:
I think this is the only type needed now.
```suggestion
export interface MlHasPrivilegesResponse {
  hasPrivileges?: estypes.SecurityHasPrivilegesResponse;
  upgradeInProgress: boolean;
}
```

Reply:
Updated in a5f1e38d10a00ceb90618472eb28da8b472b0c78